### PR TITLE
emergency fix for say

### PR DESCRIPTION
**Pull request by xf8b**

### CODE REVIEW

**Comment by xf8b** [comment]
probably some jank code but idk

**Comment by xf8b** [comment]
🤔  one sec

File: src/main/java/com/technovision/technobot/commands/fun/CommandSay.java [hunk 1]
```diff
@@ -3,19 +3,25 @@
...
 
 public class CommandSay extends Command {
-
     public CommandSay(final TechnoBot bot) {
         super(bot, "say", "Send a message using the bot", "{prefix}say [channel] <message>", Category.FUN);
     }
 
     @Override
     public boolean execute(MessageReceivedEvent event, String[] args) {
+        if (event.getMessage().mentionsEveryone()
```
Comment (Sparky200) [inline src/main/java/com/technovision/technobot/commands/fun/CommandSay.java:19]: Are you sure mentionsEveryone() works when the author doesn't have everyone permissions? This needs to be tested.

**Comment by xf8b** [comment]
bruh moment intellij reformatted

**Comment by xf8b** [comment]
re review

**Pull request merged**